kvstore/dbr/dbt.py

```diff
@@ -0,0 +1,26 @@
+from uuid import uuid4
+
+
+class DBT:
+    
+    """
+    A query spins up one or more DBTs (DB Tasks) for query execution
+    """
+    
+    def __init__(self, query=None, location=None):
+        """
+        Initialize a DBTask
+        """
+        self.id = uuid4()
+        self.dbr = None
```

Comment:
proposing that these upwards pointers don't exist, or at least are not object copies but instead dbr_ids